db/migrate/20171010172724_add_docker_manifest_list.rb

```diff
@@ -20,7 +20,7 @@ def up
       t.timestamps
     end
 
-    remove_foreign_key :katello_docker_tags, :column => :docker_manifest_id
+    #remove_foreign_key :katello_docker_tags, :column => :docker_manifest_id
```

Comment:
What's this commented out?

Reply:
Hah, good question, I didn't see this before now.

Reply:
that's result of git commit -a --amend when I was changing the message with a new ticket, sorry, repushed